app/Services/PackageLoaderService.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace App\Services;
+
+class PackageLoaderService
+{
+    public static function menuPackages()
+    {
+        $packagesList = array_pluck(config('packages'), 'name');
+        foreach ($packagesList as $package) {
+            if (config()->has('packages.' . $package)) {
```

Comment:
Why do we need this `if`? It is useless. We will always have it because we iterating right through this config :)